dashboard/db/migrate/20150423215834_create_channel_tokens.rb

```diff
@@ -0,0 +1,31 @@
+class CreateChannelTokens < ActiveRecord::Migration
+
+  def up
+    create_table :channel_tokens do |t|
+      t.string :channel, null: false
+      t.references :user, null: false
+      t.references :level, null: false
+      t.index [:user_id, :level_id], unique: true
+
+      t.timestamps
+    end
+
+    big_game_template = Level.find_by_key('Big Game Template').id
+
+    # Copy 'Big Game Template' channel for each user that has one saved.
+    PEGASUS_DB[:storage_apps].where('state != "deleted"').each do |row|
+      data = JSON.parse row[:value]
+      if data['projectTemplateLevelName'] == 'Big Game Template'
+        channel = storage_encrypt_channel_id(row[:storage_id], row[:id])
+        user = user_storage_ids_table.where(id: row[:storage_id]).first
+        if user && user[:user_id]
+          ChannelToken.create!(channel: channel, user_id: user[:user_id], level_id: big_game_template)
```

Comment:
You might want to think of what might happen if this throws an exception (eg. in case there's weird data such as duplicate Big Game Template)... would you rather stop the build (and what would you do to fix it) or print out an error (that you can repair later) and keep going.

Or alternatively examine the data in prod and verify that nothing bad will happen ;)